src/hotspot/share/prims/jvmtiH.xsl

```diff
@@ -117,6 +117,7 @@ enum {
     JVMTI_VERSION_11  = 0x300B0000,
     JVMTI_VERSION_19  = 0x30130000,
     JVMTI_VERSION_21  = 0x30150000,
+    JVMTI_VERSION_25  = 0x30170000,
```

Comment:
Should this be `JVMTI_VERSION_26` if this is targeted for version 26?

Reply:
Also the hex value represents JDK 23 not 25.